spec/services/invoices/calculate_fees_service_spec.rb

```diff
@@ -237,6 +237,32 @@
         end
       end
 
+      context 'charges are pay in advance and billable metric is recurring' do
```

Comment:
RSpec/ContextWording: Context description should match /^when\b/, /^with\b/, or /^without\b/. (https://rspec.rubystyle.guide/#context-descriptions, https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ContextWording)